### PR TITLE
Update docs about the default Eip and VIP mode

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>

Please take a look. Thanks!

/assign @zheng1 @renyunkang

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
Please also include @chaunceyjiang Xingyan as a reviewer, he has helped us to implement the default EIP mode in OpenELB.

File: content/en/docs/getting-started/configuration/configure-ip-address-pools-using-eip.md [hunk 1]
```diff
@@ -46,6 +48,10 @@ The fields are described as follows:
 
 * `name`: Name of the Eip object.
 
+* `annotations`:
+
+  * `eip.openelb.kubesphere.io/is-default-eip`: Whether the current Eip object is the default Eip object. The value can be `"true"` or `"false"`. If a default Eip object exists, you do not need to specify an Eip object when creating a Service and the system automatically assigns an IP address from the default Eip object to the Service. If you do not specify an Eip object when creating a Service while no default IP address exists, the system automatically assigns an IP address from any available Eip object, if any, to the Service. For each Kubernetes cluster, you can set only one Eip object as the default Eip object.
```
Comment (chaunceyjiang) [inline content/en/docs/getting-started/configuration/configure-ip-address-pools-using-eip.md:53]: When `eip.openelb.kubesphere.io/is-default-eip` is `true`, there is no need to add additional annotations to `Service`,for example
``` 
annotations:
  lb.kubesphere. io/v1alpha1: openelb
  protocol. openelb. kubesphere. io/v1alpha1: layer2
  eip. openelb. kubesphere. io/v1alpha2: layer2
```
these annotations are not required

**Comment by Patrick-LuoYu** [comment]
Further optimization. Please take a look. Thanks!

/assign @zheng1 @renyunkang @chaunceyjiang

**Comment by Patrick-LuoYu** [comment]
image

**Comment by chaunceyjiang** [comment]
Thank you for your work.
/LGTM

File: content/en/docs/getting-started/configuration/configure-ip-address-pools-using-eip.md [hunk 2]
```diff
@@ -46,6 +48,22 @@ The fields are described as follows:
...
+    |The Service Uses OpenELB|An Eip Object Is Specified|A Default Eip Obejct Exists|A Common Eip Object Exists|IP Address Assigment|
+    |-|-|-|-|-|
+    |No|No|No|Irrelevant|Pending|
+    |No|No|Yes|Irrelevant|An IP address from the default Eip object|
+    |Yes|No|No|No|Pending|
+    |Yes|No|No|Yes|An IP adderss from a common Eip object|
+    |Yes|No|Yes|Irrelevant|An IP address from the default Eip object|
+     |Yes|Yes|Irrelevant|No|Pending|
+    |Yes|Yes|Irrelevant|Yes|An IP address from the specified Eip object|
+
```
Comment (FeynmanZhou) [inline content/en/docs/getting-started/configuration/configure-ip-address-pools-using-eip.md:66]: @Patrick-LuoYu The format seems not correct, you can preview it on this [link](https://website-av0bn79oa-openelb.vercel.app/docs/getting-started/configuration/configure-ip-address-pools-using-eip/).

image

File: content/en/docs/getting-started/usage/use-openelb-in-vip-mode.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
+---
```
Comment (FeynmanZhou) [inline content/en/docs/getting-started/usage/use-openelb-in-vip-mode.md:1]: We might need to add `Use OpenELB in VIP Mode (Beta)` to the catalog.
image

Comment (FeynmanZhou) [reply content/en/docs/getting-started/usage/use-openelb-in-vip-mode.md:1]: It was also missing under `Concepts`, preview it [here](https://website-av0bn79oa-openelb.vercel.app/docs/concepts/).
image

File: content/en/docs/getting-started/configuration/configure-ip-address-pools-using-eip.md (hunk 2 above)
Comment (Patrick-LuoYu) [reply content/en/docs/getting-started/configuration/configure-ip-address-pools-using-eip.md:66]: I previewed the article on my local machine, which showed a correct table.

image
 
Not quite sure if anything of the markdown source code is faulty.

image

**Comment by FeynmanZhou** [comment]
@Patrick-LuoYu The Vercel preview looks good now. Thanks for your updates!
/LGTM
/Approve

**Pull request merged**